components/SpeakersSection.js

```diff
@@ -22,12 +22,32 @@ export default function TicketsSection () {
   return (
     <div className='speakers-section'>
       <SectionTitle>Speakers</SectionTitle>
-      <ul>
-        {speakers.map((speaker, index) => <li>
-          <Speaker {...speaker} />
-        </li>)}
-      </ul>
+      <div className='container'>
+        <ul className='row'>
+          {speakers.map((speaker, index) => <li className="col-sm-4">
+            <Speaker {...speaker} />
+          </li>)}
+        </ul>
+      </div>
       <style jsx>{`
+        .container {
+          width: 1060px;
+          margin: 0 auto;
+        }
+        .row {
+          position: relative;
+        }
+        .row:after {
+          content: "";
+          float: none;
+          display: block;
+          clear: both;
+        }
+        .col-sm-4 {
+          width: 33.33%;
```

Comment:
Let’s do this with mobile-first approach.

Start with `width: 100%` (only 1 column, for mobile).

Start at 320px-wide viewport and resize your browser to be larger.

At one point, you might feel that you can fit in two columns. For example, this may be 600px. Then you can write:

```css
@media (min-width: 600px) {
  .col-sm-4 { width: 50%; }
}
```

And do the same for larger screens 😄